src/sentry/web/api.py

```diff
@@ -128,7 +127,7 @@ def process_event(event_type, event_manager, project, key, remote_addr, helper,
         event_filtered.send_robust(
             ip=remote_addr,
             project=project,
-            sender=sender,
+            sender=process_event,
```

Comment:
I guess I'm also changing the sender from the endpoint class to `process_event`. I can't find anyone who cares about the sender, even in `getsentry`...